src/enums.ts

```diff
@@ -0,0 +1,10 @@
+export const enum routes {
+    MainPage = 'main-page',
+    CatalogPage = 'catalog-page',
+    ProductPage = 'product-page',
+    CartPage = 'cart-page',
+}
+
+export const enum events {
+    STORE_UPDATED = 'store_updated',
+}
```

Comment:
`Routes` and `Events`, but if you have only one `const` (`STORE_UPDATED` in `events`) do you really need enum here?